AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2583,12 +2664,18 @@ - (void)setCalculatedDisplayNodeLayout:(std::shared_ptr<ASDisplayNodeLayout>)dis
 - (CGSize)calculatedSize
 {
   ASDN::MutexLocker l(__instanceLock__);
+  if (_pendingDisplayNodeLayout != nullptr) {
```

Comment:
Currently `_pendingDisplayNodeLayout->layout` cannot be nil if `_pendingDisplayNodeLayout != nullptr`